www/js/configurator.js

```diff
@@ -301,7 +302,32 @@ const app = new Vue({
 			this.hostName = "";
 			this.hullNumber = "";
 			this.leaderIP = "";
-		}
+		},
+
+        downloadLogs: function () {
+            var xhr = new XMLHttpRequest();
+            xhr.open("GET", this.scriptPath + "create-log.sh");
+            xhr.setRequestHeader('Content-Type', 'text-plain');
+
+            xhr.onreadystatechange = function () {
+                if (xhr.readyState == 4) {
+                    if ((xhr.status >= 200 && xhr.status < 300)) {
+                        app.logsPath = xhr.responseText;
+
+                        var link = document.getElementById('logsLink');
+                        var filename =`${app.logsPath}`.substr(`${app.logsPath}`.lastIndexOf('/') + 1);
+
+                        link.setAttribute('href', filename);
+                        link.setAttribute('download', filename);
+                        link.click();
+                    } else {
+                        //mb notification if load is impossible
```

Comment:
Any ideas how to do it best? As option we can do dialog window such as on "Save" buttons.
![image](https://user-images.githubusercontent.com/20268352/61186826-758a6f00-a672-11e9-891b-bd529eb68910.png)

Reply:
Move this to separate issue, please